README.md

```diff
@@ -15,12 +15,43 @@ and idata pads. The loopback block outputs the xor of `oenable` and `odata` to
 `idata`.
 
 Sections in this README:
+* [Checking out a workspace](#checking-out-a-workspace)
 * [Creating the DUH document](#creating-the-duh-document)
 * [Scala integration](#scala-integration)
 * [Wake integration](#wake-integration)
 * [Parameterizing your block](#parameterizing-your-block)
 * [Documenting your block](#documenting-your-block)
 
+## Checking out a workspace
+This repo uses [wit](git@github.com:sifive/wit.git) for workspace management.
+This section describes how to use wit to checkout a workspace with all the
+correct versions of packages that this repo depends on. To learn more about
+how to use wit see this [tutorial](https://github.com/sifive/wit/blob/0.11.x/share/doc/wit/tutorial.md).
+
+#### Environment package
+In order for wake to be able to provide the correct environments for jobs
+that need to run the above tools, you will need an environment package that
+provides wake job runners that can fulfill those environments for your
+particular system. An example is
+[environment-example-sifive](https://github.com/sifive/environment-example-sifive).
+
+### Initialize the workspace
+```
+# Create a workspace, this will also fetch all dependencies
+wit init workspace -a git@github.com:sifive/block-pio-sifive.git
```

Comment:
Since I feel like people are going to just copy and paste this directly, can we put in the branch name (`preonboarded`) in the wit init step?